pkg/pillar/cmd/downloader/downloader.go

```diff
@@ -333,7 +333,13 @@ func maybeRetryDownload(ctx *downloaderContext,
 	status.RetryCount++
 	errStr := fmt.Sprintf("Retry attempt %d after %s",
 		status.RetryCount, status.OrigError)
```

Comment:
I think this can be deleted and just use OrigError since you have the retry_condition.

Reply:
Done